### PR TITLE
Qf-2008 provider view for employer requests 

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

**Review by dashton82 — APPROVED** [review]
Looks good - but unsure as to why we need to persist course/standard information but I'm guessing its out of the scope of this PR 😁

File: src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetAggregatedEmployerRequests/GeAggregatedtEmployerRequestsQueryResult.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetAggregatedEmployerRequests/GeAggregatedtEmployerRequestsQueryResult.cs]: File name typo

File: src/SFA.DAS.RequestApprenticeTraining.Domain/Entities/EmployerRequest.cs [hunk 2]
```diff
@@ -19,5 +19,6 @@ public class EmployerRequest
         public Guid RequestedBy { get; set; }
         public Status Status { get; set; }
         public Guid ModifiedBy { get; set; }
+        public virtual Standard Standard { get; set; }
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Domain/Entities/EmployerRequest.cs]: What is the purpose of virtual here?

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RequestApprenticeTraining.Domain/Entities/EmployerRequest.cs (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.RequestApprenticeTraining.Domain/Entities/EmployerRequest.cs]: I've removed the virtual

File: src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetAggregatedEmployerRequests/GeAggregatedtEmployerRequestsQueryResult.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.RequestApprenticeTraining.Application/Queries/GetAggregatedEmployerRequests/GeAggregatedtEmployerRequestsQueryResult.cs]: Fixed

**Review by ChrisJWoodcock — APPROVED** [review]

**Comment by ChrisJWoodcock** [comment]
This has been superseded by https://github.com/SkillsFundingAgency/das-rat-api/pull/16

**Pull request closed without merging**